eth2near/contract_wrapper/src/lib.rs

```diff
@@ -8,4 +8,7 @@ pub mod eth_client_contract_trait;
 pub mod file_eth_client_contract;
 pub mod near_contract_wrapper;
 pub mod sandbox_contract_wrapper;
-pub mod utils;
\ No newline at end of file
+pub mod utils;
+pub mod eth_network_enum;
+pub mod near_network_enum;
```

Comment:
I think `enum` suffix is redundant. To be removed in future PRs.

Reply:
fixed: https://github.com/aurora-is-near/rainbow-bridge/pull/825/commits/d817c36c4d6151b2535e1d2fd541011d6e542284